group_member_post.go

```diff
@@ -0,0 +1,35 @@
+package zoom
+
+import "fmt"
+
+// AddMenbersPath - v2 path for add group members
+const AddMenbersPath = "/groups/%s/members"
+
+// AddMemberInfo are details about a member
+type AddMemberInfo struct {
+	ID    string `json:"id"`
+	Email string `json:"email"`
+}
+
+// AddMemberOptions are details about add group members
+type AddMemberOptions struct {
+	GroupID string   `json:"-"`
+	Members []Member `json:"members"`
+}
+
+// AddMembers calls POST /groups/{groupId}/members
+func AddMembers(opts AddMemberOptions) (Member, error) {
+	return defaultClient.AddMembers(opts)
+}
+
+// AddMembers calls POST /groups/{groupId}/members
+// https://marketplace.zoom.us/docs/api-reference/zoom-api/groups/groupmemberscreate
+func (c *Client) AddMembers(opts AddMemberOptions) (Member, error) {
+	var ret = Member{}
```

Comment:
I think this should return `[]Member` not `Member`, right?

Reply:
Sorry, this func should return theese response.

```go
type ResopnseAddGroupMembers struct {
	Ids		 string `json:"ids"`
	AddedAt  string `json:"added_at"`
}
```

See this api doc response.
ref: https://marketplace.zoom.us/docs/api-reference/zoom-api/groups/groupmemberscreate